### DISCUSSION TITLE
Pydantic V2 feat Hypothesis

### DISCUSSION THREAD

**Discussion by vianmixtkz** [pydantic/pydantic#5979] [Pydantic V2]
Is Pydantic V2  compatible with Hypothesis ?
And if so are the previous limitations with constrained fields handled now ?

The limitations that I have on top of my mind:
* classes inheriting from ConstrainedStr needed to be registered in` pydantic.types._registered`
* constraints of constrained list where simply not taken into account

**Comment by Kludex** [comment]
- Not yet, you can follow the development on this issue: https://github.com/pydantic/pydantic/issues/4682
